apps/tests/integration/integration_test.go

```diff
@@ -219,154 +240,153 @@ func TestUsernamePassword(t *testing.T) {
-	}
-
-	// 3. Same scope and assertion should return cached access token
-	result2, err := cca.AcquireTokenOnBehalfOf(ctx, result.AccessToken, ccaScopes)
-	if err != nil {
-		t.Fatalf("TestOnBehalfOf: on AcquireTokenOnBehalfOf() silent token retrieval: got err == %s, want err == nil", errors.Verbose(err))
-	}
-	if result1.AccessToken != result2.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens don't match")
-	}
-
-	// 4. scope2 should return new token
-	scope2 := []string{"https://graph.windows.net/.default"}
-	result3, err := cca.AcquireTokenOnBehalfOf(ctx, result.AccessToken, scope2)
-	if err != nil {
-		t.Fatalf("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got err == %s, want err == nil", errors.Verbose(err))
-	}
-	if result3.AccessToken == "" {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got AccessToken == '', want AccessToken != ''")
-	}
-	if result3.AccessToken == result2.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens match when they should not")
-	}
-
-	// 5. scope2 should return cached token
-	result4, err := cca.AcquireTokenOnBehalfOf(ctx, result.AccessToken, scope2)
-	if err != nil {
-		t.Fatalf("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got err == %s, want err == nil", errors.Verbose(err))
-	}
-	if result4.AccessToken == "" {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got AccessToken == '', want AccessToken != ''")
-	}
-	if result4.AccessToken != result3.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens don't match")
-	}
-
-	// 6. New user assertion should return new token
-	pca1, err := public.New(pcaClientID, public.WithAuthority(organizationsAuthority))
-	if err != nil {
-		panic(errors.Verbose(err))
-	}
-	result5, err := pca1.AcquireTokenByUsernamePassword(
-		ctx, pcaScopes, user.Upn, user.Password,
-	)
-	if err != nil {
-		t.Fatalf("TestOnBehalfOf: on AcquireTokenByUsernamePassword(): got err == %s, want err == nil", errors.Verbose(err))
-	}
-	result6, err := cca.AcquireTokenOnBehalfOf(ctx, result5.AccessToken, scope2)
-	if err != nil {
-		t.Fatalf("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got err == %s, want err == nil", errors.Verbose(err))
-	}
-	if result6.AccessToken == "" {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): got AccessToken == '', want AccessToken != ''")
-	}
-	if result6.AccessToken == result4.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens match when they should not")
-	}
-	if result6.AccessToken == result3.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens match when they should not")
-	}
-	if result6.AccessToken == result2.AccessToken {
-		t.Fatal("TestOnBehalfOf: on AcquireTokenOnBehalfOf(): Access Tokens match when they should not")
-	}
-}
+// func TestConfidentialClientWithSecret(t *testing.T) {
+// 	if testing.Short() {
+// 		t.Skip("skipping integration test")
```

Comment:
Could you please leave the test uncommented and add this t.Skip instead?

Reply:
Done